microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -0,0 +1,96 @@
+---
+description: How to detect Microsoft Edge data with User-Agent Client Hints and User-Agent strings.
+title: Detecting Windows 11 using User-Agent Client Hints
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/15/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, user-agent string, ua string, user-agent client hints, user agent client hints, ua client hints, ua ch, windows 11, win11, detecting windows 11, windows detection
+---
+# Detecting Windows 11 using User-Agent Client Hints
+
+<!--
+Restrict the lexicon to these forms:
+User-Agent string
+user agent string
+User-Agent Client Hints
+-->
+
+Websites can use User Agent information that's sent from the browser to detect information such as browser brand, version, and platform. There are two approaches for sites to access User Agent information:
```

Comment:
When not specifically referring to the User-Agent Client Hints proposal, we should not capitalize "user agent" as it's not a proper name. i.e. in these two sentences it should not be capitalized.